department-of-veterans-affairs/vets-api
# Api 26185 526 v2 526 schema doc updates 2

## Summary
Updates the 526 schema in development in line with the ticket [API-26185](https://vajira.max.gov/browse/API-26185).

## Related issue(s)
[API-26185](https://vajira.max.gov/browse/API-26185)

## Testing done
- RSpec
- Visual testing to verify
- Steps to test
   * Pull down the changes
   *  run the rails server and visit `http://localhost:3000/services/claims/docs/swagger/index.html`
   * Change the docs to v2 in the dropdown in the upper right 
   * scroll down to the 526 POST request
   * Inside that area click to view the schema/example

## What areas of the site does it impact?
V2 Docs in development for the 526 form POST request.
   * modules/claims_api/config/schemas/v2/526.json - Updated in line with the ticket
   * modules/claims_api/config/schemas/v2/request_bodies/disability_compensation/request.json - Updated to match changes in schema
   * modules/claims_api/spec/fixtures/v2/veterans/disability_compensation/form_526_json_api.json -  Updated to match changes in schema
   * modules/claims_api/spec/support/swagger_shared_components/v2.rb - updated to set the request.json to be shown instead of 526.json.

## Acceptance criteria


## Requested Feedback
